### PR TITLE
Psql docker

### PR DESCRIPTION
**Pull request by pearlkwok**

### CODE REVIEW

**Comment by lijeff20001** [comment]
I think it's okay.

File: linux_sql/psql_docker.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+if [ $# -gt 2 ] || [ $# -lt 1 ]
+then
+	echo "Invalid number of arugments"
+	echo "Usage: ./psql_docker.sh start|stop [postgres_password]"
+	exit 1
+fi
+
+if [ $1 == "start" ]
+then
+	#su centos
```
Comment (WilliamxWang) [inline linux_sql/psql_docker.sh:13]: since it is hashed out might as well remove su centos

File: linux_sql/psql_docker.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	jpCreate=`docker container ls -a -f name="jrvs-psql" | wc -l`
+		
+	
+	#if the jrvs-psql is not created, create one
+	if [ $jpCreate -lt 2 ]
+	then
+		docker run --name jrvs-psql -e POSTGRES_PASSWORD=$PGPASSWORD -d -v pgdata:/var/lib/postgresql/data -p 5432:5432 postgres
+	fi
+
+	#check if `jrvs-psql` container is running
```
Comment (WilliamxWang) [inline linux_sql/psql_docker.sh:41]: maybe same as above.  consider remove it .

**Review by WilliamxWang — COMMENTED** [review]
rest is great. very good code style. i might need to do some changes to my code on coding style as well :)

**Pull request closed without merging**